js/ui/camera.js

```diff
@@ -296,7 +296,8 @@ class Camera extends Evented {
      * Pans and zooms the map to contain its visible area within the specified geographical bounds.
      *
      * @memberof Map#
-     * @param {LngLatBoundsLike} bounds The bounds to fit the visible area into.
+     * @param {LngLatBoundsLike} bounds The bounds to fit the visible area into. If bounds contains
+     *     only one point, the map center transitions to that point at zoom level options.maxZoom or {@link Map#getMaxZoom}.
```

Comment:
Neat! I didn't know that `Map#fitBounds` did this. Should we expand this parameter's type signature to `LngLatBoundsLike | LatLngLike`?